lib/resources/example_json/r4_examples_care_team.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_CARE_TEAM_ENTRY ||= {
+      'resourceType': 'CareTeam',
+      'id': 'CT-119078440-34768489-17490349-4048128-0-22554767',
+      'meta': {
+        'versionId': 1_584_980_008,
+        'lastUpdated': '2020-03-23T16:13:28Z'
+      },
+      'text': {
+        'status': 'generated',
+        'div': '<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Care Team</b></p><p><b>Patient</b>'\
+        ': FHIR, FIRE3</p><p><b>Care Team Name</b>: Psychology;Blue Team</p><p><b>Status</b>: Active</p>'\
+        '<p><b>Category</b>: Encounter-focused care team</p><p><b>Encounter</b>: 34768489</p><p><b>'\
+        'Participant</b>:</p><ul><li>Name:Dickey, Bo; Role:Psychologist (occupation)</li></ul></div>'
```

Comment:
does our code not put a space after Name: and Role: ? we'll need to fix that if so

Reply:
it doesn't

Reply:
cool. then we'll need to update this example :)

Reply:
updated https://github.com/cerner/fhir.cerner.com/pull/470/commits/d035801ecfcbf522dfa51be6884f1011b582c086#diff-f7c10b53dd7ead0281cf4dbd1281a12cR15